nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -1072,7 +1081,16 @@ in {
           phpPackage = phpPackage;
           phpEnv = {
             NEXTCLOUD_CONFIG_DIR = "${datadir}/config";
-            PATH = "/run/wrappers/bin:/nix/var/nix/profiles/default/bin:/run/current-system/sw/bin:/usr/bin:/bin";
+            PATH = lib.makeBinPath (
+              [
+                "/run/wrappers"
+                "/nix/var/nix/profiles/default"
+                "/run/current-system/sw"
+                "/usr"
+                "" # adds "/bin" to PATH
+              ]
```

Comment:
What I meant with

> but I'd very much prefer to have one of extraPackages or systemPackages for that.

was to either use these paths xor extraPackages for everything.

Given we have an agreement to go for extraPackages, I'd suggest:

* we add a warning to the release notes about the removal of these paths in 25.11 (to the 25.05 release notes)
* remove this section again
* and use `extraPackages` everywhere, as mentioned above.